IPs/RSKIP143.md

```diff
@@ -0,0 +1,42 @@
+# Segwit Compatibility
+
+|RSKIP          |143          |
+| :------------ |:-------------|
+|**Title**      |Segwit Compatibility|
+|**Created**    |22-OCT-19 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft|
+
+## Abstract
+
+This document describes the procedure allows the 2wp proccess to accept segwit transaction sent using a P2SH-PWPKH wallet. 
+
+## Motivation
+
+The 2wp proccess only allows transaction who has been sent using a P2PK address. This RSKIP describes how Bridge could accept P2SH-PWPKH[1] since this is one of the most popular wallet nowadays and the segwit transactions needs less space in a block. 
```

Comment:
The 2wp ~~proccess~~ process **currently accepts only** transaction**(s)** ~~who~~ which have been sent using a P2PK **(P2PK or P2PKH?)** address. This RSKIP describes how **the** Bridge could accept P2SH-PWPKH **(P2PKH?)**[1] since this is one of the most popular wallet**(s)** nowadays and ~~the~~ segwit transactions ~~needs~~ **use** less space in a block.